tests/models/colqwen2/test_modeling_colqwen2.py

```diff
@@ -283,7 +283,7 @@ def setUp(self):
 
     def tearDown(self):
         gc.collect()
-        torch.cuda.empty_cache()
+        backend_empty_cache(torch_device)
```

Comment:
ok for me, but we are going for `cleanup` from `src/transformers/testing_utils.py`, like

```
    def tearDown(self):
        super().tearDown()
        # clean-up as much as possible GPU memory occupied by PyTorch
        cleanup(torch_device, gc_collect=True)
```